google-home-community.groovy

```diff
@@ -432,27 +434,27 @@ private deviceTraitPreferences_Brightness(deviceTrait) {
 
 @SuppressWarnings('UnusedPrivateMethod')
 def deviceTraitPreferences_CameraStream(deviceTrait) {
-    googleCameraStreamSupportedProtocols = [
-        "progressive_mp4":         "Progressive MP4",
-        "hls":                     "HLS",
-        "dash":                    "Dash",
-        "smooth_stream":           "Smooth Stream",
-    ]
     section("Stream Camera") {
         input(
-            name: "${deviceTrait.name}.cameraStreamAttribute",
+            name: "${deviceTrait.name}.cameraStreamURLAttribute",
             title: "Camera Stream URL Attribute",
             type: "text",
-            defaultValue: "settings",
+            defaultValue: "streamURL",
             required: true
         )
         input(
-            name: "${deviceTrait.name}.cameraStreamSupportedProtocols",
-            title: "Camera Stream Supported Protocols",
-            type: "enum",
-            options: googleCameraStreamSupportedProtocols,
-            multiple: true,
-            required: true,
+            name: "${deviceTrait.name}.cameraStreamProtocolAttribute",
```

Comment:
Previous implementation only permitted a single camera protocol since it was based in the app.  Changes moved this selection to the driver to allow a multitude of camera protocol types which is breaking backwards compatibility in this case.

Reply:
Impacts the above concern as well

Reply:
User would need to select the camera protocol from the driver.

Reply:
Need to break backward compatibility to move the URLs to the driver level.